src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_001_02_Export.scala

```diff
@@ -160,7 +161,11 @@ class Pain008_001_02_Export extends LazyLogging {
     val UltmtCdtr = None
     val DbtrAgt = pain008_001_02.BranchAndFinancialInstitutionIdentificationSEPA3(pain008_001_02.FinancialInstitutionIdentificationSEPA3(DataRecord[String](None, Some("BIC"), bic)))
     val DbtrAgtAcct = None
-    val Dbtr = pain008_001_02.PartyIdentificationSEPA2(kontoDaten.nameAccountHolder.getOrElse("accountHolder subscriptor").slice(0, 70), None)
+    //solution found here: https://stackoverflow.com/questions/3322152/is-there-a-way-to-get-rid-of-accents-and-convert-a-whole-string-to-regular-lette
+    val nameAccountHolder = kontoDaten.nameAccountHolder.getOrElse("accountHolder subscriptor").slice(0, 70)
+    val normalizedNameAccountHolder = Normalizer.normalize(nameAccountHolder, Normalizer.Form.NFD)
+    val nameAccountHolderWithoutAccents = normalizedNameAccountHolder.replaceAll("[^\\p{ASCII}]", "");
```

Comment:
I know it removes special characters not part of ASCII, but isn't that a bit too strict? I think pain 008 character set is ISO-8859-1 encoded as UTF-8, so a subset of UTF-8.

Reply:
yes, however, apparently, some banks are not accepting things like ü. So, I was being safe getting rid of every special sign. This only happens at the sepa xml export and it will match with the sepa response so there is no risk there. The invoices and all data in OO will be in the original UTF8 encoding